qa/automation/Alleva 2405/Sign_in_to_app_with_Google_account.feature

```diff
@@ -0,0 +1,15 @@
+Feature: 1.1. Sign in to App with Google account
+
+Scenario: Sign in with Google account granted
```

Comment:
each scenario consists of a list of keywords in the right order according to cucumber docs